src/main/java/org/prebid/server/bidder/ix/IxBidder.java

```diff
@@ -384,4 +401,19 @@ private ObjectNode resolveBidExt(Integer duration) {
                 .video(ExtBidPrebidVideo.of(duration, null))
                 .build());
     }
+
+    private List<BidderBid> extractIxBids(BidRequest bidRequest, BidResponse bidResponse) {
+        return bidResponse == null || CollectionUtils.isEmpty(bidResponse.getSeatbid())
+                ? Collections.emptyList()
+                : bidsFromResponse(bidResponse, bidRequest, new ArrayList<>());
+    }
+
+    private List<FledgeAuctionConfig> extractFledge(BidResponse bidResponse) {
```

Comment:
I have to say that's not gonna work as expected because the fledge config for IX should be taken from the `ext.protectedAudienceAuctionConfigs`, not from the `ext.prebid.fledge.auctionconfig`

please check the PubmaticBidder to get an example and please cover it with the tests appropriately

Reply:
It's kinda strange, since it means that IX responses doesn't follows ORTB spec with their responses.

Reply:
Yeah, looks like I should revert this to previous version, that was based on PubmaticBidder.

Reply:
Done.